src/proof_of_work/config.cairo

```diff
@@ -0,0 +1,13 @@
+const MIN_PROOF_OF_WORK_BITS: u256 = 30;
+const MAX_PROOF_OF_WORK_BITS: u256 = 50;
+
+#[derive(Drop, Copy)]
+struct ProofOfWorkConfig {
+    // Proof of work difficulty (number of bits required to be 0).
+    n_bits: u8,
+}
+
+fn proof_of_work_config_validate(config: ProofOfWorkConfig) {
```

Comment:
Maybe implement as method for the struct rather than an independent function? In other configs validation is implemented like this, it would be good to have it standardized.